love/src/components/MainTel/MTDome/MTDomeShutter.jsx

```diff
@@ -82,6 +88,18 @@ export default class MTDomeShutter extends Component {
       positionCommandedShutter,
     } = this.props;
 
+    // The display considers left and right shutters as seen from above the dome.
+    const leftShutterPositionActual = Math.abs(positionActualShutter[1]);
+    const rightShutterPositionActual = Math.abs(positionActualShutter[0]);
+    const leftShutterPositionActualPixels = -Math.floor(leftShutterPositionActual * widthShutterInPixels);
+    const rightShutterPositionActualPixels = Math.floor(rightShutterPositionActual * widthShutterInPixels);
+
+    // The display considers left and right shutters as seen from above the dome.
+    const leftShutterPositionCommanded = Math.abs(positionCommandedShutter[1]);
+    const rightShutterPositionCommanded = Math.abs(positionCommandedShutter[0]);
+    const leftShutterPositionCommandedPixels = Math.floor(leftShutterPositionCommanded * widthShutterInPixels);
```

Comment:
On line 94 there is a `-` symbol but not here? Should there be one here as well or should the one on line 94 be removed?

Reply:
Good catch! thank you @wvreeven. Indeed, line 100 also required a `-` symbol and I missed that. I will change this to add `-` to the SVG attribute itself so it's more clear that we are moving pixels in the negative direction.